addons/project/project.py

```diff
@@ -686,6 +686,12 @@ def onchange_user_id(self, cr, uid, ids, user_id, context=None):
             vals['date_start'] = fields.datetime.now()
         return {'value': vals}
 
+    def onchange_date_deadline(
+            self, cr, uid, ids, date_end, date_deadline, context=None):
+        if not date_end or (date_end[:10] == self.browse(
```

Comment:
browsing a record in an onchange will not work for records not created yet, I don't see the point of checking the value in db anyway.